spec/app/ics-030-middleware/README.md

```diff
@@ -101,6 +108,12 @@ function onChanOpenInit(
         counterpartyChannelIdentifier,
         appVersion, // note we only pass app version here
     )
+    abortTransactionUnless(err != nil)
```

Comment:
If we call `abortTransactionUnless` here, is there a reason to return `Error`?

Reply:
This is true, but I'm just trying to stay consistent with the ICS26 function signature. In practice, the abortTransaction process is carried out by returning an error at this level.